code/__DEFINES/turfs.dm

```diff
@@ -94,3 +94,8 @@
 #define TURF_PATHING_PASS_PROC 1
 /// Turf is never passable
 #define TURF_PATHING_PASS_NO 2
+
+/// Generate horizontal stripes for turf decals
+#define STRIPE_HORIZONTAL 0
+/// Generate vertical stripes for turf decals
+#define STRIPE_VERTICAL 1
```

Comment:
Should mention these are only used for holiday coloring

Reply:
Sure.